base-models/pom.xml

```diff
@@ -45,6 +45,11 @@
 			<artifactId>base62</artifactId>
 			<version>${seruco.base62.version}</version>
 		</dependency>
+		
+		<dependency>
```

Comment:
We only need a dependency for the SimpleGrantedAuthority class, this should be enough:
    <dependency>
      <groupId>org.springframework.security</groupId>
      <artifactId>spring-security-config</artifactId>
    </dependency>

The spring-boot-starter-security dependency drags in a lot more other dependencies that are not needed in base-models module.